demo/src/main/AndroidManifest.xml

```diff
@@ -20,6 +19,10 @@
       </intent-filter>
     </activity>
 
+    <meta-data
+      android:name="com.novoda.downloadmanager.MaxConcurrentDownloads"
+      android:value="1" />
```

Comment:
@blundell we've added 2 flavours of the demo project, one serial and one parallel. This way the user can select which flavour to run 8a29785